src/OrchardCore/OrchardCore.Abstractions/OrchardCore.Abstractions.csproj

```diff
@@ -17,6 +17,7 @@
     <PackageReference Include="Microsoft.Extensions.Localization" />
     <PackageReference Include="Microsoft.Extensions.Logging" />
     <PackageReference Include="Newtonsoft.Json" />
+    <PackageReference Include="NodaTime" />
```

Comment:
Nope

Reply:
I was worried when I added, but we need to expose `CalendarSystem`, let review again and come back to this one

Reply:
I think it's fine to remove this because I already move the types to the `OrchardCore.Localization` namespace